specificatie/datum.yaml

```diff
@@ -55,12 +59,16 @@ components:
             onbekend:
               type: boolean
               default: true
+            leesbaarFormaat:
+              type: string
       required:
         - onbekend
+        - leesbaarFormaat
       example:
         value:
           type: DatumOnbekend
           onbekend: true
+          leesbaarFormaat: "Onbekend"
```

Comment:
in de features is onbekend met kleine letter "o" geschreven

Reply:
Aangepast

Reply:
@JohanBoer je zegt "Aangepast", maar er staat nog steeds langFormaat: "Onbekend"